src/@dvcorg/gatsby-theme-iterative/components/Documentation/RightPanel/index.tsx

```diff
@@ -207,14 +208,14 @@ const RightPanel: React.FC<IRightPanelProps> = ({
             Have a question? Join our chat, we will help you:
           </p>
 
-          <Link
+          <SmartLink
```

Comment:
I think a standard `a` would be better here since we know this link will always be one, but it's not that big a deal.

Reply:
Didn't notice earlier, that we had a prop to opt-out of pre-redirect. So, revert back to Link with the `optOutPreRedirect` prop.